cmd/internal/cli/actions_linux.go

```diff
@@ -129,7 +130,7 @@ func execStarter(cobraCmd *cobra.Command, image string, args []string, name stri
 
 	engineConfig := singularityConfig.NewConfig()
 
-	configurationFile := buildcfg.SYSCONFDIR + "/singularity/singularity.conf"
+	configurationFile := files.GetSysConfigFile()
```

Comment:
there's the `syfs` package in `pkg/syfs`. I think that's a better place for this code.

Reply:
I put it there because it is not user-facing but internal.

Reply:
Having two packages that deal with the layout of the singularity files is confusing. There's some pending clean up that needs to happen in terms of what's available to other packages (if that's what you mean by "user facing") but I would rather have one problem pending of a fix and not two.

Reply:
Well, issues are there for that. I would prefer to do the right thing here, i.e., put the package in the right place. I guess we need to see if we can find consensus about this because on who review my PRs, it is once one way, once the other way and this is not good to change code based on what appears to me to be personal preferences. In other words, let's set some guidelines.

Reply:
The rule is pretty simple here, if the package use `buildcfg` constants, it's an internal package because it would be unusable as public package otherwise.

An other approach could be to add some get accessor in `buildcfg` package to have things like `buildcfg.GetSingularityConfigFile()`

Reply:
I think there is agreement that the current location is adequate. We also have issue #3995 to finalize the discussion and agree on the potential next steps.

Reply:
Agreement is something else.

I hadn't realized that you sneaked the change into a different unrelated code review. You could have said so.